### ISSUE TITLE
Adapting to Speed: The Flexibility of Frame Rates in Real-Time Systems

### ISSUE THREAD

**Issue by CristoJV** [ElenaRyumina/EMO-AffectNetModel#2]
Hi Elena Ryumina 👋! 

First off, congratulations on your incredible work! 
I'm genuinely impressed by what you've achieved. I have a couple of questions that piqued my curiosity. 

🤔 Does your system accommodate different frame rates, or is it optimized for a fixed frame rate? Also, I'm wondering if it operates in real-time? 🕒

Thank you so much for your time and effort!

Looking forward to your response. 
Keep up the fantastic work! 👍

**Comment by ElenaRyumina** [comment]
Hi @CristoJV,

Thank you for your kind feedback on our work. I apologize for the delayed reply.

Regarding FPS, we trained our LSTM models on datasets containing recordings with frame rates ranging from 7 to 60 FPS. In order to preserve the contextual information of the videos, we downsampled the frame rate of all videos to 5 FPS by thinning frames at a fixed rate depending on the original frame rate of the video. As a result, the LSTM models were trained on sequences of 10 frames, or two seconds. However, you have the option to run the models without downsampling frames and consider alternative durations other than two seconds. However, this setup may affect accuracy.

For real-time processing, the prediction time of the models is affected by several factors, including the required frame rate, the duration of the video segment, the face detector used, and the specifications of your personal computer or device. You have the flexibility to tailor the model's operation to your needs, ensuring a satisfactory balance between detection speed and accuracy.

Thank you once again for your review. I hope that our development proves to be useful for you.
